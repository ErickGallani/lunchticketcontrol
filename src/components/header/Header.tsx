import * as React from 'react';
import * as PropTypes from 'prop-types';
import FlatButton from 'material-ui/FlatButton';
import * as styles from './Header.css';

export default class Header extends React.Component {
    constructor(props: any){
        super(props);
    }
        
    render() {
        return (
            <div className={styles.title}>
                <p>Header</p>
            </div>
        );
    }
}