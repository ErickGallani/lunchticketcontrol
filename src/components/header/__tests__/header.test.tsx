import * as React from 'react';
import Header from "../Header";
import { shallow, mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders correctly headr', () => {
  const elem = shallow(<Header title="test" />);
  expect(shallowToJson(elem)).toMatchSnapshot();
});