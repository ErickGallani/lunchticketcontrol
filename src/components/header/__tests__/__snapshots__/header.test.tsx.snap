// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly headr 1`] = `
<div
  className="title"
>
  <p>
    Header
  </p>
</div>
`;
